### PR TITLE
Add automation for BZ:1994654/SAT-4845

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Dependant PR 
https://github.com/SatelliteQE/airgun/pull/1003
https://github.com/SatelliteQE/airgun/pull/1006

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate
airgun: 1003

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate
airgun: 1003

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1302,12 +1302,23 @@ def test_global_registration_form_populate(
...
         assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
         assert module_lce.name in cmd['advanced']['life_cycle_env_helper']
         assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
 
+        new_org = target_sat.api.Organization().create()
+        new_ak = target_sat.api.ActivationKey(organization=new_org).create()
+        session.organization.select(org_name=new_org.name)
+        cmd = session.host.get_register_command(
+            {
+                'general.orgnization': new_org.name,
```
Comment (adarshdubey-star) [inline tests/foreman/ui/test_host.py:1315]:
```suggestion
                'general. organization': new_org.name,
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/ui/test_host.py]: Good catch, @adarshdubey-star !
It was just copy pasta from existing test :P Updated it now, thanks!

And this typo still exists in other tests which are working as organization field gets autoselected selected when org is selected at top, so no one caught this yet!  
It was fixed in airgun https://github.com/SatelliteQE/airgun/pull/835 long ago but author missed updating in robottelo for tests, so will open a separate PR for it as it'll require to cherrypick to older versions

Comment (Gauravtalreja1) [reply tests/foreman/ui/test_host.py]: https://github.com/SatelliteQE/robottelo/pull/12902

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate
airgun: 1003

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate
airgun: 1006

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k test_global_registration_form_populate
airgun: 1006

**Review by sambible — APPROVED** [review]

**Review by shubhamsg199 — APPROVED** [review]
Ack

**Pull request merged**